docs/overview.rst

```diff
@@ -215,9 +215,21 @@ deployed in specific regions, select that region from the menu. You can also
 create a new project for the organization in the selected region by clicking
 the *Create project* button in the top right.
 
-The cog icon next to each project takes you to the Project Settings page, in
-the menu simply called Settings, as :ref:`described further below
-<overview-project-settings>`.
+If you have previously created a `CrateDB Edge region`_, the *Create project*
+menu will also allow you to set a custom backup location for any cluster
+deployed within your new project. You can do this by defining the name, the
+key, and the secret of the backup location in the project creation menu:
+
+.. image:: _assets/img/projects-create.png
+   :alt: Cloud Console projects creation menu
+
+The *Test connection* button allows you to test the connection to the custom
```

Comment:
it might be worth noting that when a custom backup location is selected, the form cannot be submitted until the connection to the backup bucket has been verified